backend/dal/Services/BaseService`.cs

```diff
@@ -0,0 +1,31 @@
+using System.Security.Claims;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Services
+{
+    /// <summary>
+    /// BaseService abstract class, provides a generic service layer to perform CRUD operations on the datasource.
+    /// </summary>
+    /// <typeparam name="ET"></typeparam>
+    public abstract class BaseService<ET> : BaseService where ET : BaseEntity
```

Comment:
Doesn't provide much functionality presently, but will make it quicker to build generic service implementations.